### PR TITLE
Create a separate client application for MVP 0.4 features development

### LINKED ISSUE #327: Create a separate client application for MVP 0.4 features development
### Overview
Set up a new React application for MVP 0.4 features development on the client side.

### Action Items

- [x] Create a separate folder on the project root level and name it `client-mvp-04`.
- [x] Set up a new React application.
- [x] Apply Sass setup from #223 implementation.
- [x] Set up the new client local deployment from the VRMS root folder.

### PR DESCRIPTION
**Pull request by OlgaBilogurova**
Fixes #327

### CODE REVIEW

File: client-mvp-04/nginx/default.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+server {
+
+    listen 3000;
```
Comment (nickbeaird) [inline client-mvp-04/nginx/default.conf]: Are we expecting to have two frontend containers? If that is the case, then we will need to put this on a different port, or else we will have a collision. If so, how about 3001.

File: client-mvp-04/package.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "local-storage": "^2.0.0",
+    "mathjs": "^6.6.2",
+    "minimist": "^1.2.3",
+    "moment": "^2.24.0",
+    "moment-recur": "^1.0.7",
+    "node-sass": "^4.14.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.3",
+    "typescript": "^4.0.3"
```
Comment (nickbeaird) [inline client-mvp-04/package.json]: Are we writing this new frontend in Typescript?

File: client-mvp-04/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (nickbeaird) [inline client-mvp-04/.eslintrc.json:1]: 👍

File: client-mvp-04/package.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  },
+  "scripts": {
+    "start": "dotenv -e .env -e ../backend/.env -- cross-var cross-env PORT=%CLIENT_PORT% react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --watchAll=false",
+    "test:watch": "react-scripts test",
+    "eject": "react-scripts eject",
+    "heroku-postbuild": "npm run build"
```
Comment (nickbeaird) [inline client-mvp-04/package.json]: You can remove the heroku-postbuild

**Review by nickbeaird — CHANGES_REQUESTED** [review]

File: client-mvp-04/nginx/default.conf (hunk 1 above)
Comment (OlgaBilogurova) [reply client-mvp-04/nginx/default.conf]: I'm not sure that we'll run two clients simultaneously. Anyway, this makes sense. I'll change the port to 3001.

File: client-mvp-04/package.json (hunk 2 above)
Comment (OlgaBilogurova) [reply client-mvp-04/package.json]: No, just JS. I tried to resolve all warnings and set it up. Thank you for catching it. It's better to uninstall unuseful dependency. 👍

File: client-mvp-04/package.json (hunk 4 above)
Comment (OlgaBilogurova) [reply client-mvp-04/package.json]: Ok

File: client-mvp-04/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# VRMS - MVP 0.4
+
+VRMS MVP 0.4 - a client application for MVP 0.4 features development.
+
+## Technologies
+
+- [React](https://reactjs.org/docs/getting-started.html)
+- [Redux](https://redux.js.org/introduction/getting-started)
+- [SASS](https://sass-lang.com/guide)
```
Comment (alex-anakin) [inline client-mvp-04/README.md]: Please add testing technologies here.

File: client-mvp-04/README.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+You should now have a live app.
+
+## Running Tests
+
+To run all `client-mvp-04` tests:
+
+- Navigate to the `client-mvp-04` directory
+- Run `npm test` or `npm run test`
+
+You should now see all existing tests are passing.
```
Comment (alex-anakin) [inline client-mvp-04/README.md]: tests are passing -> tests passed

File: client-mvp-04/Dockerfile.client [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+FROM node:14.11.0-slim AS client-builder
+USER node
+WORKDIR /srv/client
+COPY --from=client-development /srv/client/node_modules node_modules
+COPY . .
+USER root
+RUN npm run build
+
+FROM nginx as client-production
+EXPOSE 3000
```
Comment (alex-anakin) [inline client-mvp-04/Dockerfile.client]: 3000 -> 3001

**Review by alex-anakin — CHANGES_REQUESTED** [review]

File: client-mvp-04/README.md (hunk 5 above)
Comment (OlgaBilogurova) [reply client-mvp-04/README.md]: Sure.

File: client-mvp-04/README.md (hunk 6 above)
Comment (OlgaBilogurova) [reply client-mvp-04/README.md]: Thanks

**Review by alex-anakin — APPROVED** [review]

**Pull request merged**